src/screens/ProfileScreen/index.js

```diff
@@ -281,10 +281,9 @@ const ProfileScreen = ({route}) => {
     try {
       setIsFetchingList(true);
       setIsHitApiFirstTime(true);
-      const cacheFeed = StorageUtils.myFeeds.get();
       const result = await getSelfFeedsInProfile(offset, limit);
       const {data: dataMyFeed} = result;
-      const {mapNewData} = mappingColorFeed({dataFeed: dataMyFeed, dataCache: cacheFeed});
+      const mapNewData = mappingColorFeed(dataMyFeed);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code removes the usage of cached feeds in the `mappingColorFeed` function. This could potentially affect performance if fetching feeds from the server is slow or if there are network issues. If the cache was used to improve performance, consider reintroducing it.
```diff
- const result = await getSelfFeedsInProfile(offset, limit);
- const {data: dataMyFeed} = result;
- const mapNewData = mappingColorFeed(dataMyFeed);
+ const cacheFeed = StorageUtils.myFeeds.get();
+ const result = await getSelfFeedsInProfile(offset, limit);
+ const {data: dataMyFeed} = result;
+ const {mapNewData} = mappingColorFeed({dataFeed: dataMyFeed, dataCache: cacheFeed});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
Closed